custom/enikshay/ucr/data_sources/person_for_referral_report_v2.json

```diff
@@ -1323,6 +1322,254 @@
+                                             "treatment_after_failure",
+                                             "treatment_after_lfu",
+                                             "other_previously_treated"
+                                         ]
+                                     },
+                                     "expression_if_true": {
+                                         "type": "constant",
+                                         "constant": "Previously Treated"
+                                     },
+                                     "expression_if_false": {
+                                         "type": "constant",
+                                         "constant": ""
+                                     }
+                                 }
+                             }
+                         },
+                         "expression_if_false": {
+                             "type": "constant",
+                             "constant": ""
+                         }
+                     }
+                 }
+            },
+            {
+                "display_name": "date_of_diagnosis",
+                "transform": {},
+                "datatype": "date",
+                "type": "expression",
+                "is_primary_key": false,
+                "is_nullable": true,
+                "column_id": "date_of_diagnosis",
+                "expression": {
+                    "type": "nested",
+                    "argument_expression": {
+                        "type": "enikshay_episode_from_person",
+                        "person_id_expression": {
+                            "type": "property_name",
+                            "property_name": "_id"
+                        }
+                    },
+                    "value_expression": {
+                        "type": "property_name",
+                        "property_name": "date_of_diagnosis"
+                    }
+                }
+            },
+            {
+                "display_name": "episode_type",
+                "transform": {},
+                "datatype": "string",
+                "type": "expression",
+                "is_primary_key": false,
+                "is_nullable": true,
+                "column_id": "episode_type",
+                "expression": {
+                    "type": "nested",
+                    "argument_expression": {
+                        "type": "enikshay_episode_from_person",
+                        "person_id_expression": {
+                            "type": "property_name",
+                            "property_name": "_id"
+                        }
+                    },
+                    "value_expression": {
+                        "type": "property_name",
+                        "property_name": "episode_type"
+                    }
+                }
+            },
+            {
+                "display_name": "adherence_schedule_id",
```

Comment:
same as above. i dont think this meets the spec. It should be `schedule_daily` -> `Daily`, `schedule_mwf` -> `Intermittent (MWF)`, and `schedule_trs` -> `Intermittent (TTS)`